README.md

```diff
@@ -218,11 +218,21 @@ brew install curl cmake libressl c-ares
 
 #### Windows (using vcpkg)
 
+Probably the easiest way to download and compile `google-cloud-cpp` on Windows
+is to use [vcpkg][vcpkg-link], a package manager from Microsoft that downloads
+the source code for a library (and all its dependencies), compiles it, and adds
+the right configuration for DevStudio and/or CMake.
+
+After downloading and compiling `vcpkg`, simply run:
```

Comment:
I may have not said this before, but refrain from using words such as "easiest" and "simple". Comfortable developers may find this path straight forward, but new developers might hit headaches.

Reply:
Done.